test/e2e/main_test.go

```diff
@@ -253,6 +253,27 @@ func TestManualMeteringInstall(t *testing.T) {
 			},
 			MeteringConfigManifestFilename: "node-selector-prometheus-importer-disabled.yaml",
 		},
+		{
+			Name:                      "ValidHDFS-MySQLDatabase",
+			MeteringOperatorImageRepo: meteringOperatorImageRepo,
+			MeteringOperatorImageTag:  meteringOperatorImageTag,
+			Skip:                      false,
+			PreInstallFunc:            createMySQLDatabase,
+			InstallSubTests: []InstallTestCase{
+				{
+					Name:     "testReportingProducesData",
+					TestFunc: testReportingProducesData,
+					ExtraEnvVars: []string{
+						"REPORTING_OPERATOR_PROMETHEUS_DATASOURCE_MAX_IMPORT_BACKFILL_DURATION=15m",
+						"REPORTING_OPERATOR_PROMETHEUS_METRICS_IMPORTER_INTERVAL=30s",
+						"REPORTING_OPERATOR_PROMETHEUS_METRICS_IMPORTER_CHUNK_SIZE=5m",
+						"REPORTING_OPERATOR_PROMETHEUS_METRICS_IMPORTER_INTERVAL=5m",
+						"REPORTING_OPERATOR_PROMETHEUS_METRICS_IMPORTER_STEP_SIZE=60s",
+					},
+				},
+			},
+			MeteringConfigManifestFilename: "mysql.yaml",
```

Comment:
Open to suggestions on any additional post-installation functions. I didn't see much value in adding a post-install function that aims to emulate something like the prometheus connector up test, as we should be able to tell if the Hive metastore + mysql connection is working with the `testReportingProducesData` sub-test.